src/graphql/queries/__tests__/__snapshots__/ListReplies.js.snap

```diff
@@ -44,15 +65,55 @@ Object {
     "ListReplies": Object {
       "edges": Array [
         Object {
+          "highlight": Object {
+            "hyperlinks": Array [],
+            "reference": null,
+            "text": "<HIGHLIGHT>bar</HIGHLIGHT>",
+          },
           "node": Object {
             "id": "userFoo",
           },
         },
         Object {
+          "highlight": Object {
+            "hyperlinks": Array [],
+            "reference": null,
+            "text": "<HIGHLIGHT>bar</HIGHLIGHT>",
+          },
           "node": Object {
             "id": "rumor",
           },
         },
+        Object {
+          "highlight": Object {
+            "hyperlinks": Array [],
+            "reference": "<HIGHLIGHT>http</HIGHLIGHT>://gohome.com",
```

Comment:
This makes me notice that we [forgot to use `cjk_url_email` analyzer](https://github.com/cofacts/rumors-db/blob/master/schema/replies.js#L8) for `reference` field......
This is a bug that will affect reply search. I am adding a ticket for this.